tools/umi_tools/main.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.preview.dsl = 2
+
+// Local default params
+params.internal_process_name = 'umitools_dedup'
```

Comment:
Other than naming the output directory, what is this parameter doing? In terms of the output structure I personally prefer `umitools/dedup` if there are multiple commands.

Reply:
I'd prefer to drop if it's only doing one thing.. 👍

Reply:
No nothing else - if you remember I developed a groovy class for parameter inheritance from the calling nf file. In our internal modules, we can override this parameter by setting `params.umitools_dedup_process_name`. I removed this functionality for nf-core as Paulo regarded it as a hack and now this param is kinda dangling as a result